source/nvda.pyw

```diff
@@ -10,6 +10,7 @@
 import sys
 import os
 import globalVars
+import ctypes

```

Comment:
The only way we could allow this would be that if nvda.pyw is executed outside of the virtual environment that we then execute a subprocess of runnvda or runnvdaw.bat depending on whether python or pythonw was used to originally execute it. But I think that starts getting into some possible recursion danger. I feel it is probably better and safer not to allow executing like this from source anymore, without using runnvda or runnvdaw.